src/Symfony/Component/Form/Resources/translations/validators.sq.xlf

```diff
@@ -68,11 +68,11 @@
             </trans-unit>
             <trans-unit id="112">
                 <source>Please select a valid language.</source>
-                <target state="needs-review-translation">Ju lutem zgjidhni një gjuhë të vlefshme.</target>
+                <target>Ju lutemi zgjidhni një gjuhë të vlefshme.</target>
             </trans-unit>
             <trans-unit id="113">
                 <source>Please select a valid locale.</source>
-                <target>Ju lutemi zgjidhni një lokale të vlefshme.</target>
+                <target>Ju lutemi zgjidhni një “locale” të vlefshme.</target>
```

Comment:
Is it required to use the double quotes? I don't think we use it elsewhere. What if we remove them?

Reply:
Great question 😅 
[Ardian Vehbiu](https://en.wikipedia.org/wiki/Ardian_Vehbiu) has written an entire "[small book](https://peizazhe.com/2020/03/05/drejtshkrimi-i-emrit-te-huaj)" trying to tackle what is the correct way to write foreign names in Albanian. 

Short Context: 

Albanian has not been updated since 1980 iirc. There were attempts to do so in 2007 and 2013, but all that amounted on was restructuring the entities that oversee this process. The only news I can find for a new update is this article that [a new dictionary will be published in 2024](https://euronews.al/fjalori-i-gjuhes-shqipe-do-te-pasurohet-me-70-mije-fjale-te-reja/).

Why is this important? Because a lot of computer science words do not exist in Albania officially and officially there is a grammar rule how to write foreign names or words. They are written how they are pronounced, with Albanian phonetics. 

Example: `file` is just how you would say `friend` in Greek, in Albanian. Now if you used `fajl` or `fajll` depending on the area it would be correct. Luckily, `file -> skedë` already exists. 

For `locale` is it even weirder, because `lokale` can mean either multiple of bar, café, pub and so on, or local but only for public functions. Like local elections. So the phonetic rule does not applay. 

Writing `locale` only now means that the person reading this has to understand a) this is a foreign word b) read it in its foreign pronunciation  (not important) and c) know the word (situational)

Now, there is a convention on writing foreign names or words in papers. [§120 at this Wiki](https://sq.wikipedia.org/wiki/Thonj%C3%ABzat) by using the original language's script as long as it is quoted. We know can even add accents for words in French or Italian 👍 

I have always used this, and I am not the only one doing it, when doing translation work where a) the name is a prosoponym or b) a registered mark (“Apple” vs Epëll or Ejpëll) or c) a foreign word in a specific context. 

And I purposefully used `“”` for this so it does not collide with `""`

PS. I can remove it, but it just looks sloppy :/

Reply:
I updated this part, and as it was discussed over Symfony Slack I applied a convention for both files. To no lose this, I checked and saw that [XLIFF 1.2 supports note nodes](http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#file). `file > header > note` 

The note is in Albanian, because it seemed like the proper helper for Albanian people contributing, but I can add an english version to it to. What is says is the following: 

```
For foreign words, whom have no direct translation, please observe the following rules: 
a) when the noun is an acronym widely used as a prosoponym, then, the name is cased without quotation and the suffix is written with a dash. The gender is decided ad hoc. Example: JSON-i (masculine)
b) when the noun is an acronym not widely used as a prosoponym, hen, the name is cased without quotation and the suffix is written with a dash. The gender is feminine. Example: URL-ja (feminine)
c) when the name is in definite form because of the case in the sentence, then, the name is cased without quotation and the suffix is written with a dash. Example: host-i, prej host-it
d) when the name is in indefinite form because of no case in the sentence, then, the name written surrounded with quotation marks “”. Example: “locale”
```

I added `a)` and `b)` because there is a distinction on how acronyms and initialisms are gendered in Albanian. And `c` and `d` covers when to use quotations and when a dash is enough.

Reply:
P.S. I wanted to apologize if this is looking a bit pedantic, but my reasoning is that by having sloppy grammar / translations when using Symfony, it just makes all of Symfony look and feel sloppy to people who use it or pitch with it for projects.